lib/AST/Builtins.cpp

```diff
@@ -1932,6 +1945,19 @@ static ValueDecl *getHopToActor(ASTContext &ctx, Identifier id) {
   return builder.build(id);
 }
 
+static ValueDecl *getPackCount(ASTContext &ctx, Identifier id) {
+  BuiltinFunctionBuilder builder(ctx, /* genericParamCount */ 1,
+                                 /* anyObject */ false,
+                                 /* areParametersPack */ true);
+
+  auto packEx = makeGenericParam(/* index */ 0, /* isPackExpansion */ true);
```

Comment:
It seems a bit odd to have an isPackExpansion flag here. Why not handle this like metatypes and tuples, and add a `makePackExpansion()` utility?